zalando-incubator/solution-center-header
# fix(icons): fix svg icons in firefox

Fixes #126 

Test:
- confirm user and help icons appear in Chrome, Safari, and Firefox
- confirm hover effect (orange)
